README.md

```
@@ -1,4 +1,166 @@
+This branch contains elements of  `discord-summarizer-bot`, another bot designed to provide concise summaries of channel conversations using OpenAI's GPT-4. 
+
+
+## Features
+
+* Summarizes Discord channel messages in response to user commands or a REST API call.
+* Using its scheduler, can summarize future or past periods
+* Uses OpenAI's GPT-4 for accurate and coherent summaries.
+* Easy to deploy and use within any Discord server.
+
+Interactive Use
+* TODO: outline summary_bot's features as listed on https://top.gg/bot/1058568749076185119
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)